geant4.spec

```diff
@@ -1,13 +1,13 @@
-### RPM external geant4 11.1.2
+### RPM external geant4 11.1.ref07
 ## INCLUDE compilation_flags
 ## INCLUDE compilation_flags_lto
 %define use_vecgeom 0
-#%define tag 35f304cde19a3e2dfe3bb941d4fc76c2fe45fa26
+%define tag d35b0306a837c2b3640b250c1af599688eee8df1
 #%define branch cms/v%{realversion}
-#%define github_user cms-externals
-%define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
-%define branch master
-%define github_user Geant4
+%define github_user cms-externals
+#%define tag d35b0306a837c2b3640b250c1af599688eee8df1
+#%define branch master
+#%define github_user Geant4
```

Comment:
please remove all these commented  out code ( this also break automatic forward ports)